bin/ofs/src/fuse.rs

```diff
@@ -636,6 +637,107 @@ impl PathFilesystem for Fuse {
             entries: relative_paths.chain(children).skip(offset as usize).boxed(),
         })
     }
+    async fn rename2(
+        &self,
+        req: Request,
+        origin_parent: &OsStr,
+        origin_name: &OsStr,
+        parent: &OsStr,
+        name: &OsStr,
+        _flags: u32,
+    ) -> Result<()> {
+        log::debug!(
+            "rename2(origin_parent={:?}, origin_name={:?}, parent={:?}, name={:?})",
+            origin_parent,
+            origin_name,
+            parent,
+            name
+        );
+        self.rename(req, origin_parent, origin_name, parent, name)
+            .await
+    }
+
+    async fn lseek(
+        &self,
+        _req: Request,
+        path: Option<&OsStr>,
+        fh: u64,
+        offset: u64,
+        whence: u32,
+    ) -> Result<ReplyLSeek> {
+        log::debug!(
+            "lseek(path={:?}, fh={}, offset={}, whence={})",
+            path,
+            fh,
+            offset,
+            whence
+        );
+
+        let file = self.get_opened_file(FileKey::try_from(fh)?, path)?;
+
+        if !file.is_read && !file.is_write {
+            Err(Errno::from(libc::EACCES))?;
+        }
+
+        let whence = whence as i32;
+
+        let offset = if whence == libc::SEEK_CUR || whence == libc::SEEK_SET {
```

Comment:
I didn't get this. Do we need to use `offset` + `cur` while `SEEK_CUR`?

Reply:
Yeah, you are right, we need to save `offset` in `opendedfile` context, which will do in the future.

Reply:
I have flag this situation as `ENOSYS`

Reply:
done, PTAL